packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {TabPrimitive} from '@twilio-paste/tabs-primitive';
+import {TabsContext} from './TabsContext';
+
+type Orientation = 'horizontal' | 'vertical' | undefined;
+
+const getTabBoxStyles = (orientation: Orientation): {} => {
+  switch (orientation) {
+    case 'vertical':
+      return {
+        display: 'block',
+        right: '-2px',
+        paddingBottom: 'space40',
+        paddingTop: 'space40',
+        paddingLeft: 'space50',
+        paddingRight: 'space50',
+        borderRightWidth: 'borderWidth20',
+        borderRightStyle: 'solid',
+        borderRightColor: 'transparent',
+        _selected: {
+          borderRightColor: 'colorBorderPrimary',
+        },
+      };
+    case 'horizontal':
+    default:
+      return {
+        display: 'inline-block',
+        bottom: '-2px',
+        paddingBottom: 'space10',
+        paddingTop: 'space20',
+        paddingLeft: 'space50',
+        paddingRight: 'space50',
+        borderBottomWidth: 'borderWidth20',
+        borderBottomStyle: 'solid',
+        borderBottomColor: 'transparent',
+        _selected: {
+          borderBottomColor: 'colorBorderPrimary',
+        },
+      };
+  }
+};
+
+const getTabTextStyles = (props: TabProps): {} => {
+  if (props['aria-disabled']) {
+    return {color: 'colorTextWeaker'};
+  }
+  if (props['aria-selected']) {
+    return {
+      color: 'colorText',
+      _hover: {
+        color: 'colorText',
+      },
+    };
+  }
+  return {
+    color: 'colorTextWeak',
+    _hover: {
+      color: 'colorText',
+    },
+  };
+};
+
+export interface TabProps {
```

Comment:
If you import `TabPrimitiveProps` and extend it here, I think that will fix the TS error here.

Reply:
Ignore that. Still having errors.